nomad/server.go

```diff
@@ -893,8 +893,6 @@ func (s *Server) setupRPC(tlsWrap tlsutil.RegionWrapper) error {
 		return err
 	}
 
-	s.logger.Printf("[INFO] nomad: RPC listening on %q", s.rpcListener.Addr().String())
-
```

Comment:
why did this go away?

Reply:
this comment: https://github.com/hashicorp/nomad/pull/3670#discussion_r166793930